### PR TITLE
Resolve merge conflicts in resource_category_spec.rb

### LINKED ISSUE #23: resource_category_spec: Missing validation unit tests

### PR DESCRIPTION
**Pull request by joeyst**
closes #23

### CODE REVIEW

File: spec/models/resource_category_spec.rb [hunk 1]
```diff
@@ -44,4 +44,43 @@
...
+        let (:resource_category) { ResourceCategory.new }
+        it "has a name" do
+            expect(resource_category).to_not be_valid
+            expect(resource_category).to validate_presence_of(:name)
+            resource_category.name = 'resource category'
+            expect(resource_category).to be_valid
+        end
+
+        it "has a valid name when length 1 or greater" do
+            expect(resource_category).to_not be_valid
```
Comment (LukeEff) [inline spec/models/resource_category_spec.rb:57]: I think this line does not need to exist

File: spec/models/resource_category_spec.rb [hunk 2]
```diff
@@ -44,4 +44,43 @@
...
+            resource_category.name = ''
+            expect(resource_category).to_not be_valid
+            resource_category.name = 'a'
+            expect(resource_category).to be_valid
+        end
+
+        it "has a valid name when length 255 or lower" do
+            expect(resource_category).to_not be_valid
```
Comment (LukeEff) [inline spec/models/resource_category_spec.rb:65]: This also seems unnecessary

File: spec/models/resource_category_spec.rb [hunk 3]
```diff
@@ -44,4 +44,43 @@
 
     end
 
+    describe "validations" do
```
Comment (LukeEff) [inline spec/models/resource_category_spec.rb:47]: Ruby uses 2 space indentation

**Review by LukeEff — CHANGES_REQUESTED** [review]
Good to merge after some minor changes

File: spec/models/resource_category_spec.rb (hunk 3 above)
Comment (joeyst) [reply spec/models/resource_category_spec.rb:47]: Alright, I'll change the indentation. Might have to do it after class

**Review by LukeEff — APPROVED** [review]
Looks great :)

**Pull request merged**